src/plugins/simpleini/simpleini.c

```diff
@@ -126,8 +249,17 @@ int elektraSimpleiniGet (Plugin * handle, KeySet * returned, Key * parentKey)
 	}
 
 	char * key = 0;
+	char * strippedkey = 0;
 	char * value = 0;
 	int errnosave = errno;
+	
+	char * format = getReadFormat (handle);
+	if (!format)
+	{
+		ELEKTRA_SET_ERROR (ELEKTRA_ERROR_NOEOF, parentKey, "invalid 'format' specified");
```

Comment:
Maybe a general error for "invalid plugin configuration" is enough for such cases?